.github/workflows/python-publish.yml

```diff
@@ -57,8 +57,38 @@ jobs:
         with:
           path: dist/*.tar.gz
 
-  upload_pypi:
+  test_wheels:
     needs: [build_wheels, build_sdist]
+    name: Run tests on wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 2
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/download-artifact@v2
+        name: Download artifacts
+        with:
+          name: artifact
+          path: dist
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.6'
+
+      - name: Install wheel and tests dependencies
+        run: |
+          pip install $(echo dist/*manylinux2010_x86_64.whl)[tests]
+
+      - name: Run tests
+        run: |
+          tar xf dist/*.tar.gz --wildcards wbia-pyflann*/test/ && \
+          cd wbia-pyflann*/test && \
+          pytest
```

Comment:
nifty! Do you know if all the unit and doctests are running as a result of this command?

Reply:
No the doctests are not run as a result of this command.  I've added an extra command `xdoctest` to the latest commit.